src/main/scala/mesosphere/marathon/core/base/Clock.scala

```diff
@@ -0,0 +1,15 @@
+package mesosphere.marathon.core.base
+
+import mesosphere.marathon.state.Timestamp
+
+trait Clock {
```

Comment:
Sure. Where to?

Basically, I just create a directory structure for new stuff.